### ISSUE TITLE
Firefox: cannot tab into Lookup with a selected item

### ISSUE THREAD

**Issue by minevskiy** [salesforce/design-system-react#205]
When lookup has a pill inside (i.e. an item is selected), I cannot tab into it in Firefox 44.

Repro steps in Demo app:
1. Open Modal demo https://design-system-react-stage.herokuapp.com/#/modal
2. Click "New Opportunity"
3. Tab into "Account Name" lookup and select any item
4. Tab out and try to tab back into the lookup.
